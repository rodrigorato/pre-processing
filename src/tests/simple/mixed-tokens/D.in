alias a0 = "#{args[0]}";
public class Foo {
	alias display = System.out.println;
    public void foo(String[] args) {
    	display(#a0);
    }
}