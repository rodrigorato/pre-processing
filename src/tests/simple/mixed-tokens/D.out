
public class Foo {
	
    public void foo(String[] args) {
    	System.out.println("" + (args[0]) + "");
    }
}